src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
+@import '../utils/variables';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
   margin: 0;
+  padding: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-columns;
+  max-width: (($day-size + $gap-columns) * $columns - $gap-columns);
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    height: $day-size;
+    width: $day-size;
+    border: 1px solid black;
+    animation: translateY(-20px);
```

Comment:
It seems like you're trying to use an animation named 'translateY', but you haven't defined any keyframes for it. Ensure you define the keyframes for the 'translateY' animation or remove this line if it's not needed.